integrations/malware_tools_analyzers/Dockerfile

```diff
@@ -103,6 +103,24 @@ RUN wget -q https://github.com/mandiant/GoReSym/releases/download/v2.7.4/GoReSym
     && unzip GoReSym-linux.zip \
     && chmod +x GoReSym \
     && ln -s ${PROJECT_PATH}/goresym/GoReSym /usr/local/bin/goresym
+# Install DroidLysis
+WORKDIR "${PROJECT_PATH}/droidlysis"
+RUN python3 -m venv venv \
+ && . venv/bin/activate \
+ && mkdir -p ~/softs \
+ && cd ~/softs \
+ && wget -q https://bitbucket.org/iBotPeaches/apktool/downloads/apktool_2.9.3.jar \
+ && wget -q https://bitbucket.org/JesusFreke/smali/downloads/baksmali-2.5.2.jar \
+ && wget -q https://github.com/pxb1988/dex2jar/releases/download/v2.4/dex-tools-v2.4.zip \
+ && unzip dex-tools-v2.4.zip \
+ && rm -f dex-tools-v2.4.zip \
+ && apt-get -y install --no-install-recommends openjdk-17-jdk \
+ && pip3 install --no-cache-dir git+https://github.com/cryptax/droidlysis \
+ && mkdir -p "${PROJECT_PATH}/droidlysis/out" "${PROJECT_PATH}/droidlysis/conf" \
+ && DROIDLYSIS_PATH="$(pip3 show droidlysis | grep Location | cut -d ' ' -f 2)" \
+ && cp "${DROIDLYSIS_PATH}/conf/"*.conf "${PROJECT_PATH}/droidlysis/conf/" \
+ && rm -f "${PROJECT_PATH}/droidlysis/conf/general.conf"
+COPY ./droidlysis/general.conf /opt/deploy/droidlysis/conf/general.conf
```

Comment:
why you do all of this processing for the general.conf file? Are there other configuration files that needs to be maintained there? If yes, which one?
Otherwise you could just mount the general.conf file as a volume in the docker compose file. That would also allow the user to change that configuration at runtime

Reply:
Yes there are many other config files. They are all required. General conf files needs to be unique wrt our system and the rest can be as it is.